pkg/clients/frontegg_client.go

```diff
@@ -127,7 +127,20 @@ func getToken(ctx context.Context, password string, endpoint string) (string, st
 
 	email, ok := claims["email"].(string)
 	if !ok {
-		return "", "", time.Time{}, errors.New("email claim not found in token")
+		// If email is not present (service account case), use metadata.user or sub as identifier
+		if metadata, hasMetadata := claims["metadata"].(map[string]interface{}); hasMetadata {
+			if user, hasUser := metadata["user"].(string); hasUser {
+				email = user
+			}
+		}
+
+		if email == "" {
```

Comment:
Is `email` guaranteed to be an empty string when `claims["email"]` doesn't return `ok` as `true`? If not, wondering if we should add an `email = ""` right before `if metadata, hasMetadata := claims["metadata"].(map[string]interface{}); hasMetadata {`

Reply:
Ah yes, this is a good point! Thanks Jun!